### PR TITLE
Move environment related tasks into cibox-env role

### PR DESCRIPTION
**Pull request by Sanchiz**
Note: *Tested in locally for Vagrant and CI server.*

### CODE REVIEW

File: roles/cibox-env/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+- name: Recover from possible fail
+  sudo: yes
+  shell: 'dpkg --configure -a'
+  ignore_errors: yes
+
+- name: Update apt cache
+  sudo: yes
+  apt: update_cache=yes
+  ignore_errors: yes
```
Comment (podarok) [inline roles/cibox-env/tasks/main.yml]: two tasks above should be in pre_tasks folder

File: roles/cibox-env/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- name: Setup initial packages onto clean system for ansible
+  sudo: yes
+  # Installing sudo on small ubuntu install.
+  raw: "apt-get -y install python-simplejson sudo curl"
+
+- name: Fixing logger not found
+  sudo: yes
+  shell: "apt-get --reinstall install -y bsdutils"
+
+# We have to disable mail sending from CI box.
```
Comment (podarok) [inline roles/cibox-env/tasks/main.yml]: Let's rephrase it a bit
```Disabling mail sending from CI server```

File: roles/cibox-env/tasks/main.yml (hunk 1 above)
Comment (Sanchiz) [inline roles/cibox-env/tasks/main.yml]: Roles do not support `pre_tasks`.

Comment (Sanchiz) [inline roles/cibox-env/tasks/main.yml]: https://github.com/ansible/ansible/issues/3869

Comment (podarok) [inline roles/cibox-env/tasks/main.yml]: ok, let's use your workaround for now. ignore my comment above

**Pull request merged**